modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,60 @@
+
+== {KialiProduct} architecture
+
+Kiali is based on the open source link:https://kiali.io/[Kiali project]. Kiali is composed of two components: the Kiali application and the Kiali console.
+
+Kiali Server (back end):: This component runs in the container application platform and communicates with the service mesh components, retrieves and processes data, and exposes this data to the console. The Kiali application does not need storage. When deploying the application to a cluster, configurations are set in ConfigMaps and secrets.
+
+Kiali console (front end):: The Kiali console is a web application. 	The Kiali application serves the Kiali console, which then queries the Kiali Server for data to present it to the user.
+
+In addition, Kiali depends on external services and components provided by the container application platform and Istio.
+
+Red Hat Service Mesh (Istio):: Istio is a Kiali requirement. Istio is the component that provides and controls the service mesh. Although Kiali and Istio can be installed separately, Kiali depends on Istio and will not work if it is not present. Kiali needs to retrieve Istio data and configurations, which are exposed through Prometheus and the cluster API.
+
+Prometheus:: A dedicated Prometheus instance is included as part of the {SMProductName} installation. When Istio telemetry is enabled, metrics data are stored in Prometheus. Kiali uses this Prometheus data to determine the mesh topology, display metrics, calculate health, show possible problems, and so on. Kiali communicates directly with Prometheus and assumes the data schema used by Istio Telemetry. Prometheus is an Istio dependency and a hard dependency for Kiali, and many of Kiali's features will not work without Prometheus.
+
+Cluster API:: Kiali uses the API of the {product-title} (cluster API) to fetch and resolve service mesh configurations. Kiali queries the cluster API to retrieve, for example, definitions for namespaces, services, deployments, pods, and other entities. Kiali also makes queries to resolve relationships between the different cluster entities. The cluster API is also queried to retrieve Istio configurations like virtual services, destination rules, route rules, gateways, quotas, and so on.
+
+Tracing:: Tracing is optional, but when you install {DTProductName} and Kiali is configured, the Kiali console includes a tab to display distributed tracing data. Note that tracing data will not be available if you disable Istio’s distributed tracing feature. Also note that the user must have access to the namespace where the Istio control plane is installed to view tracing data.
+
+Grafana:: Grafana is optional. When available, the metrics pages of Kiali display links to direct the user to the same metric in Grafana. Note that user must have access to the namespace where the Istio control plane is installed to view links to the Grafana dashboard and view Grafana data.
```

Comment:
> Note that user must have access to the namespace where the Istio control plane is installed to view links to the Grafana dashboard and view Grafana data.

Again, I'm not sure this is true. If it is, it is news to me :) @josunect  can confirm.

Reply:
Yes, you can see the links even if the user doesn't have access to the control plane: 
![image](https://github.com/user-attachments/assets/8379262c-cd4a-44cf-b589-ed6736369985)
![image](https://github.com/user-attachments/assets/ba4d5ee9-3750-4fe4-b31e-d14ec7a8ba9d)

With an user with access, you can see the same links: 
![image](https://github.com/user-attachments/assets/cef26b75-4a20-4f90-8074-3e9760989a42)